Add TODO to openQA documentation (#4469)
os-autoinst/openQA: docs/UsersGuide.asciidoc

=== Filtering test results and builds ===

At the top of the test results overview page is a form which allows filtering tests by result,

> architecture and TODO-status. TODO-status means that tests still require review.

Reviewer: ```suggestion
architecture and TODO-status. TODO means that tests still require review.
```

Additionally, "review" could be a link leading to the "Review badges" section where the term "review" is clarified. (Maybe there's a more fitting target? I just thing that "still require review" is also a bit vague.)

Author: I agree that it is kind of vague, althoug the Review badge section also just escribes "being reviewed" as "a bug reference or a test issue reason is stated"

image::images/filter_form.png[Filter form]

There is also a similar form at the bottom of the index page which allows filtering builds by

> group and customizing the limits. This also allows filtering by TODO-status

Reviewer 2: This is not correct. The form relates to the filter form at the bottom of the *index* page, e.g. openqa.opensuse.org/ on the bottom. This is not the same as the *tests* page. Better write

```suggestion
group and customizing the limits.
Also the 'All tests' table allows filtering by the TODO-status.
```

Author: Oh yeah I should have caught that, seeing as there is neither a similar form and much less one at the bottom of the All Tests page

> group and customizing the limits. This also allows filtering by TODO-status

Reviewer: ```suggestion
group and customizing the limits. The form also allows filtering by TODO-status.
```
("This" is a bit unclear in that context.)

=== Filtering test results and builds ===

At the top of the test results overview page is a form which allows filtering tests by result,
architecture and TODO-status. TODO-status means that <<#_review_badges,tests still require>> review.

image::images/filter_form.png[Filter form]

There is also a similar form at the bottom of the index page which allows filtering builds by
group and customizing the limits.

> Also the 'All tests' table allows filtering by the TODO-status.

Author: I think an argument can be made for both ways of writing this because the lower section describes places other than the tests overview page where results can be filtered.

> Also the 'All tests' table allows filtering by the TODO-status.

Reviewer: Ok, I suppose that's correct. Then do it however you like :-)
